scripts/zones/Jugner_Forest/mobs/Giollemitte_B_Feroun.lua

```diff
@@ -0,0 +1,51 @@
+----------------------------------
+-- Area: Jugner_Forest
+--   NM: Giollemitte B Feroun
+-- Involved in Quest: A Timely Visit
+-----------------------------------
+require("scripts/globals/status")
+require("scripts/globals/msg")
+-----------------------------------
+
+function onMobInitialize(mob)
+
+    mob:setMobMod(dsp.mobMod.IDLE_DESPAWN, 300) -- 5 minutes
+    mob:setMobMod(dsp.mobMod.ADD_EFFECT, 1)
+    
+end
+
+function onMobSpawn(mob)
+
+    mob:addMod(dsp.mod.SILENCERES, 80)
+    mob:addMod(dsp.mod.SLEEPRES, 50)
+    mob:addMod(dsp.mod.LULLABYRES, 50)
+    mob:addMod(dsp.mod.MDEF, 50)
+    DespawnMob(mob:getID(), 300)
+    
+end
+
+function onAdditionalEffect(mob, target, damage)
+end
+
+function onMobFight(mob, target)
+
+    if mob:getHPP() <= 25 and mob:getLocalVar("invincible") == 0 then
```

Comment:
this is handled by a mixin now
https://github.com/DarkstarProject/darkstar/blob/master/scripts/mixins/job_special.lua#L29

Reply:
Changed to use mixins.